beancount/parser/printer_test.py

```diff
@@ -89,15 +89,17 @@ def assertRoundTripViaRealFile(self, entries1, errors1, test_write_source=True):
         # Contex manager can be used as of python 3.12
         # see https://github.com/python/cpython/issues/58451
         # https://github.com/beancount/beancount/issues/222
-        oss1 = tempfile.NamedTemporaryFile("w", suffix=".beancount", delete=False)
+        oss1 = tempfile.NamedTemporaryFile(
+            "w", suffix=".beancount", delete=False, newline="\n"
+        )
 
         oss1.write('option "plugin_processing_mode" "raw"\n')
         printer.print_entries(entries1, file=oss1)
 
         oss1.close()
 
-        with open(oss1.name, "r") as file:
-            oss1_value = file.read()
+        with open(oss1.name, "rb") as file:
+            oss1_value = file.read().decode()
```

Comment:
Why this change? As far as I can tell the old and new code does exactly the same thing.